nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -175,6 +184,10 @@ public StandardProcessSession(final ProcessContext context) {
         this.sessionId = idGenerator.getAndIncrement();
         this.connectableDescription = description;
 
+        this.isRollbackCountEnabled = NiFiProperties.getInstance().isRollbackCountEnabled();
+        this.isRollbackLogUnackFFEnabled = NiFiProperties.getInstance().isRollbackLogUnackFFEnabled();
+        this.rollbackLogUnackFFMax = NiFiProperties.getInstance().getRollbackLogUnackFFMax();
+
```

Comment:
Good point.  So, is the correct handling to have a successful transfer or commit drop the attribute?